python/task_runner.py

```diff
@@ -31,12 +34,43 @@
 import cm.config as config
 import cm.job
 from cm.logger import log
-from cm.models import TaskLog, JobLog, LogStorage
+from cm.models import TaskLog, JobLog, LogStorage, ADCM, ConfigLog
 
 
 TASK_ID = 0
 
 
+def log_rotation():
+    adcm_object = ADCM.objects.get(id=1)
+    config_log = ConfigLog.objects.filter(obj_ref=adcm_object.config).latest('date')
+    adcm_config = json.loads(config_log.config)
+    log_rotation_on_db = adcm_config['job_log']['log_rotation_in_db']
+    log_rotation_on_fs = adcm_config['job_log']['log_rotation_on_fs']
+    rotation_jobs_on_db = []
+    rotation_jobs_on_fs = []
+
+    for job in JobLog.objects.all():
```

Comment:
I believe you could do a filter over finish_date with order_by and create index on finish_date. That allow you to get old much faster that iterate over all list. 

Your code now tend to be slow, because mostly this code iterate over thousands row to remove only one.